### ISSUE TITLE
@babel/runtime could not resolve

### ISSUE THREAD

**Issue by rendinjast** [nextui-org/nextui#134]
**Describe the bug**
HI.
Library need @babel/runtime and in some case it's not preinstalled, so we get an error that says could not resolve @bable/runtime.

Installing @bable/runtime fixes the error, but it should be a dependency for @nextui-org/react

**Steps to reproduce**
- yarn create vite 
- setup a React (JS) project
- install @nextui-org/react
- run dev server 

**Environment / Context**
- Operating System:  Arch Linux
- Browser and version Chrome 96.0.4664.110

**Additional info**
log error:
```
(this will be run only when your dependencies or config have changed)
 > node_modules/@nextui-org/react/esm/index.js:3:37: error: Could not resolve "@babel/runtime/helpers/interopRequireDefault" (mark it as external to exclude it from the bundle, or surround it with try/catch to handle the failure at run-time)
    3 │ var _interopRequireDefault = require("@babel/runtime/helpers/interopRequireDefault");
      ╵                                      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

 > node_modules/@nextui-org/react/esm/use-current-state/index.js:3:37: error: Could not resolve "@babel/runtime/helpers/interopRequireDefault" (mark it as external to exclude it from the bundle, or surround it with try/catch to handle the failure at run-time)
    3 │ var _interopRequireDefault = require("@babel/runtime/helpers/interopRequireDefault");
      ╵                                      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

....

error when starting dev server:
Error: Build failed with 119 errors:
node_modules/@nextui-org/react/esm/avatar/avatar-group.js:3:37: error: Could not resolve "@babel/runtime/helpers/interopRequireDefault" (mark it as external to exclude it from the bundle, or surround it with try/catch to handle the failure at run-time)
node_modules/@nextui-org/react/esm/avatar/avatar-group.styles.js:3:37: error: Could not resolve "@babel/runtime/helpers/interopRequireDefault" (mark it as external to exclude it from the bundle, or surround it with try/catch to handle the failure at run-time)
node_modules/@nextui-org/react/esm/avatar/avatar.js:3:37: error: Could not resolve "@babel/runtime/helpers/interopRequireDefault" (mark it as external to exclude it from the bundle, or surround it with try/catch to handle the failure at run-time)
node_modules/@nextui-org/react/esm/avatar/index.js:3:37: error: Could not resolve "@babel/runtime/helpers/interopRequireDefault" (mark it as external to exclude it from the bundle, or surround it with try/catch to handle the failure at run-time)
node_modules/@nextui-org/react/esm/backdrop/backdrop.js:3:37: error: Could not resolve "@babel/runtime/helpers/interopRequireDefault" (mark it as external to exclude it from the bundle, or surround it with try/catch to handle the failure at run-time)
...
    at failureErrorWithLog (/home/rend/nextui-vite/node_modules/esbuild/lib/main.js:1493:15)
    at /home/rend/nextui-vite/node_modules/esbuild/lib/main.js:1151:28
    at runOnEndCallbacks (/home/rend/nextui-vite/node_modules/esbuild/lib/main.js:941:63)
    at buildResponseToResult (/home/rend/nextui-vite/node_modules/esbuild/lib/main.js:1149:7)
    at /home/rend/nextui-vite/node_modules/esbuild/lib/main.js:1258:14
    at /home/rend/nextui-vite/node_modules/esbuild/lib/main.js:629:9
    at handleIncomingPacket (/home/rend/nextui-vite/node_modules/esbuild/lib/main.js:726:9)
    at Socket.readFromStdout (/home/rend/nextui-vite/node_modules/esbuild/lib/main.js:596:7)
    at Socket.emit (node:events:390:28)
    at addChunk (node:internal/streams/readable:324:12)
error Command failed with exit code 1.i
```

**Comment by jrgarciadev** [comment]
Hey @rendinjast  thanks for reporting it

**Comment by jrgarciadev** [comment]
Hey @rendinjast I just released version `v1.0.1-beta.2` which includes the solution for this 🙌🏻
